src/AppBundle/Controller/CustomerController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+class CustomerController extends Controller
+{
+    public function customersLoginAction(Request $request)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $apiKeyHead = $request->headers->get('API-Key-Token');
+        $facebookToken = $request->headers->get('social_token');
```

Comment:
"social_network": "string" written in the api dock, for  differents oauth  future